### DISCUSSION TITLE
Orange Pi 800

### DISCUSSION THREAD

**Discussion by davidstoll** [Joshua-Riek/ubuntu-rockchip#72] [General]
Just wondered if this would be compatible with the Opi800?  It is the same generation as the Op5.  Similar to Raspberry Pi 4 and the 400.

**Comment by Joshua-Riek** [comment]
No, it would not be compatible with the Orange Pi 800. It uses a different SOC, the RK3399, and would need a different kernel, bootloader, and device tree. Additionally, it requires different drivers and tweaks for optimal performance.

**Reply by mlanner** [reply]
Do you have any plans at all to implement a build for the OPi 800?
